tools-reference/head-and-tail/text.xml

```diff
@@ -69,6 +69,18 @@ sed -n -e '1,5{ s/foo/bar/g ; p }' input.txt &gt; output.txt
 # on line 5
 sed -n -e 's/foo/bar/gp ; 5q' input.txt &gt; output.txt
 
+# bad: replace 'foo' with 'bar', and drop first five lines
+# (this one is from a real example!)
```

Comment:
do we really need this comment here? it being real or artificial adds little value to the point it is trying to make.

Reply:
I've dropped the comment.